content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/__methods__/update_status.rb

```diff
@@ -0,0 +1,119 @@
+module Transformation
```

Comment:
@fdupont-redhat To stay consistent with other ManageIQ methods please add ManageIQ and Automate
https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/launch_ansible_job.rb#L8